js/page-specific/phone-directory/phone-directory.js

```diff
@@ -0,0 +1,131 @@
+var jQuery = $.noConflict(true);
+var pageCount = 0;
+
+function __onPrevPage() {
+    for (var i = 1; i <= pageCount; i++) {
+        if (document.getElementById("Page" + i).style.display == "block") {
+            document.getElementById("Page" + i).style.display = "none";
+            document.getElementById("Page" + (i - 1)).style.display = "block";
+            break;
+        }
+    }
+    setNextPrevious(i - 1);
+}
+
+function setNextPrevious(pageNum) {
+    if (pageNum < pageCount) {
+        document.getElementById("Next").style.display = "block";
+    } else {
+        document.getElementById("Next").style.display = "None";
+    }
+
+    if (pageNum > 1) {
+        document.getElementById("Previous").style.display = "block";
+    } else {
+        document.getElementById("Previous").style.display = "None";
+    }
+}
+
+function validateForm() {
+    var firstName = jQuery.trim(
+        document.getElementById("txtFirst").value
+    );
+    var lastName = jQuery.trim(
+        document.getElementById("txtLast").value
+    );
+    if (firstName == "" && lastName == "") {
+        alert(
+            "If Searching all Agencies, You must enter at least one letter of the last name or one letter of the first name"
+        );
+        return false;
+    } else {
```

Comment:
since if is return false, you could remove the else and just return true;

Reply:
Removed the else clause